src/index.js

```diff
@@ -1,2 +1,73 @@
 import './style.css';
-import 'bootstrap/dist/css/bootstrap.css';
\ No newline at end of file
+import 'bootstrap/dist/css/bootstrap.css';
+
+const gameID = 'OyWOyYBAfoODiap1EhZ8';
+const API_URL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores/`;
+const tbody = document.getElementById('tbody');
+const searchID = (id) => document.getElementById(id);
+
+const PostToApi = async () => {
+  const user = searchID('user').value;
+  const score = searchID('score').value;
+  const userData = { user, score };
+
+  try {
+    await fetch(API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: JSON.stringify(userData),
+    });
+  } catch (error) {
+    throw new Error(error.message);
+  }
+};
```

Comment:
[OPTIONAL] I think resetting the fields after a successful submission could improve the overall user experience. To improve upon this you could also make sure that the fields are required. Overall, very good job! :clap: